zombienet_tests/.set_env

```diff
@@ -0,0 +1,20 @@
+pathprepend() {
+  for ((i=$#; i>0; i--));
+  do
+    ARG="$@[$i]"
+    if [ -d "$ARG" ] && [[ ":$PATH:" != *":$ARG:"* ]]; then
+        PATH="$ARG${PATH:+":$PATH"}"
+    fi
+  done
+}
+
+# paths are prepend in order, so you can manage with version will run
+# by the order of this array
+CUSTOM_PATHS=(
+  "<path>/polkadot/target/release"
```

Comment:
in my case at least <path> is my home directory. I think it make sense to change `<path>` here to `~` to have a default that can work without altering this file  (at least in my case :P but I assume others have similar setups)

Reply:
Make sense, I will change it to have a nice default that can work out of the box.
Thanks!